File: javascript-closure-angular-client/API/Client/ItemSimple.js
goog.provide('API.Client.ItemSimple');

/**
 * @record
 */
API.Client.ItemSimple = function() {}

/**
 * Company ID
 * @type {!API.Client.UUID}
 * @export
 */
API.Client.ItemSimple.prototype.companyId;

/**
 * ERP Code
 * @type {!string}
 * @export
 */
API.Client.ItemSimple.prototype.code;

/**
 * Agast Code
 * @type {!string}
 * @export
 */
API.Client.ItemSimple.prototype.agast;

/**
 * Item Description
 * @type {!string}
 * @export
 */
API.Client.ItemSimple.prototype.description;

File: async-scala-client/src/main/scala/io/swagger/client/model/ItemSimple.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class ItemSimple (
  companyId: UUID,  // Company ID
code: String,  // ERP Code
agast: String,  // Agast Code
description: Option[String]  // Item Description
)
